references.html.haml

```diff
@@ -13,12 +13,34 @@ title: References
 %p 
   WildFly is a JavaEE application server which uses Infinispan as the base for its clustering needs: web sessions, stateful session beans, etc
   
-%a(name="hibernate")
+%a(name="hibernate-orm")
 %h3 Hibernate ORM
 %p
-  %a(href="http://www.hibernate.org") http://www.hibernate.org
+  %a(href="http://www.hibernate.org/orm") http://www.hibernate.org/orm
 %p
-  Hibernate ORM has a second-level cache provider built on top of Infinispan
+  Hibernate ORM has a second-level cache provider built on top of Infinispan.
+
+%a(name="hibernate-ogm")
+%h3 Hibernate OGM
+%p
+  %a(href="http://www.hibernate.org/ogm") http://www.hibernate.org/ogm
+%p
+  Hibernate OGM can perform CRUD operations on Infinispan, replacing the RDBMS.
```

Comment:
It's not very clear who does replace the RDBMS. And it neither fits OGM nor Infinispan.

"Hibernate OGM can perform CRUD operations on Infinispan, using Infinispan as a datastore." maybe? Or "persistent storage" but I'm not sure it fits well with Infinispan.